### PR TITLE
chore: Generate initial google-analytics-data-v1alpha gapic

### PR DESCRIPTION
**Pull request by dazuma**

### CODE REVIEW

File: google-analytics-data-v1alpha/lib/google/analytics/data/v1alpha/alpha_analytics_data/client.rb [hunk 1]
```diff
@@ -0,0 +1,760 @@
...
+# Auto-generated by gapic-generator-ruby. DO NOT EDIT!
+
+require "google/cloud/errors"
+require "google/analytics/data/v1alpha/analytics_data_api_pb"
+
+module Google
+  module Analytics
+    module Data
+      module V1alpha
+        module AlphaAnalyticsData
```
Comment (quartzmo) [inline google-analytics-data-v1alpha/lib/google/analytics/data/v1alpha/alpha_analytics_data/client.rb:26]: It really seems unusual and incorrect to include "Alpha" in this module name. Is there anything that can be done about this?

File: google-analytics-data-v1alpha/synth.py [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+gapic = gcp.GAPICMicrogenerator()
+library = gapic.ruby_library(
+    "analytics/data", "v1alpha",
+    proto_path="google/analytics/data/v1alpha",
+    generator_args={
+        "ruby-cloud-gem-name": "google-analytics-data-v1alpha",
+        "ruby-cloud-title": "Google Analytics Data V1alpha",
+        "ruby-cloud-description": "The Google Analytics Data API provides programmatic methods to access report data in Google Analytics App+Web properties. With the Google Analytics Data API, you can build custom dashboards to display Google Analytics data, automate complex reporting tasks to save time, and integrate your Google Analytics data with other business applications.",
+        "ruby-cloud-env-prefix": "ANALYTICS_DATA",
+        "ruby-cloud-grpc-service-config": "google/analytics/data/v1alpha/analytics_data_grpc_service_config.json",
```
Comment (quartzmo) [inline google-analytics-data-v1alpha/synth.py:33]: No `ruby-cloud-product-url` for this gem?

File: google-analytics-data-v1alpha/synth.py [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+gapic = gcp.GAPICMicrogenerator()
+library = gapic.ruby_library(
+    "analytics/data", "v1alpha",
+    proto_path="google/analytics/data/v1alpha",
+    generator_args={
+        "ruby-cloud-gem-name": "google-analytics-data-v1alpha",
+        "ruby-cloud-title": "Google Analytics Data V1alpha",
+        "ruby-cloud-description": "The Google Analytics Data API provides programmatic methods to access report data in Google Analytics App+Web properties. With the Google Analytics Data API, you can build custom dashboards to display Google Analytics data, automate complex reporting tasks to save time, and integrate your Google Analytics data with other business applications.",
+        "ruby-cloud-env-prefix": "ANALYTICS_DATA",
```
Comment (quartzmo) [inline google-analytics-data-v1alpha/synth.py:32]: Should google-analytics-admin-v1alpha and google-analytics-data-v1alpha really have different env var prefixes? `ANALYTICS_ADMIN` and `ANALYTICS_DATA`?

File: google-analytics-data-v1alpha/lib/google/analytics/data/v1alpha/alpha_analytics_data/client.rb (hunk 1 above)
Comment (dazuma) [reply google-analytics-data-v1alpha/lib/google/analytics/data/v1alpha/alpha_analytics_data/client.rb:26]: Ugh, thanks for catching that. Yes, we do have a generator switch specifically for changing bad service names like that. I'll apply it and regen...

File: google-analytics-data-v1alpha/synth.py (hunk 2 above)
Comment (dazuma) [reply google-analytics-data-v1alpha/synth.py:33]: No, no public URL yet. (We don't have one for analytics admin either.)

File: google-analytics-data-v1alpha/synth.py (hunk 3 above)
Comment (dazuma) [reply google-analytics-data-v1alpha/synth.py:32]: Hmm. I guess our closest precedent is bigtable-admin which just uses `BIGTABLE`. Maybe both should be `ANALYTICS`. Then we should probably change analytics-admin as well (now, while it's still in private alpha).

Comment (quartzmo) [reply google-analytics-data-v1alpha/synth.py]: I agree, I feel that based on precedent it should be `ANALYTICS`.

**Review by quartzmo — APPROVED** [review]

**Pull request merged**